### PR TITLE
Project 1: Initialize project with components.

### PR DESCRIPTION
**Pull request by er-contreras**
- Initialise React app.
- Add React Redux (npm install react-redux).
- Structure your application files using a "feature folder" approach and use the ducks pattern for your Redux files. Your files/folder structure could look like this:
```
/src
    |---/components
    |---/redux
        |--- /books
                | books.js
        |--- /categories
                | categories.js
        | cofigureStore.js
    | index.js
```
-The building blocks of your app should be set as re-usable components.
- Add React Router and set two <Route>s and <Link>s for the app's navigation:
Books
- Styling is not required at this point.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @eri8-9 👋

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
```
Comment (mikenath223) [inline README.md]: - [x] Kindly update the readme following this [template](https://github.com/microverseinc/readme-template).

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Books = () => (
+  <div>Books Section</div>
+);
+
```
Comment (mikenath223) [inline src/redux/books/books.js]: - [x] Kindly place the Book component in the component folder the redux folder should only contain the redux reducers and store.

- [ ] Also, the Book component is expected to receive the book value as a prop and render the author and title.

File: src/redux/categories/categories.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>Categories Section</div>
+);
+
```
Comment (mikenath223) [inline src/redux/categories/categories.js]: - [x] Same for the Category component kindly place it in the components folder.

- [ ] In addition, the category component should receive the category as a prop.

**Review by Emmyn5600 — APPROVED** [review]
Hi, 

Good job so far! 👏🏼 @eri8-9 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.
 
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**